cumulus_library/studies/core/core_templates/documentreference.sql.jinja

```diff
@@ -0,0 +1,64 @@
+{% import 'core_utils.jinja' as utils %}
+CREATE TABLE core__documentreference AS
+WITH temp_documentreference AS (
+    SELECT DISTINCT
+        {{- utils.basic_cols(
+                'documentreference',
+                'dr',
+                [
+                    'id',
+                ]
+            ) 
+        }},
+        {{- utils.nullable_cols(
+                'documentreference',
+                'dr',
+                [
+                    'type',
+                    'status',
+                    'docstatus',
+                    'context',                
+                    ('subject', 'reference', 'subject_ref'),
+                    ('context', 'period', 'start', 'author_date')
+                ], 
+                schema
+            ) 
+        }},
+        {{- utils.truncate_date_cols(
+                'documentreference',
+                'dr',
+                [
+                    ('author_date', 'day'),
+                    ('author_date', 'week'),
+                    ('author_date', 'month'),
+                    ('author_date', 'year'),
+                ],
+                schema
+            ) 
+        }},
+    FROM documentreference AS dr
+)
+
+SELECT DISTINCT
+    type_coding.type_row AS doc_type,
+    coalesce(type_coding.type_row.code, 'None') AS doc_type_code,
```

Comment:
Not `cumulus__missing-or-null`?

Reply:
This is a field that could reasonably be expected to be null, so we explictly coalesce it to avoid the error state reporting.